examples/fitness/lib/measurement.dart

```diff
@@ -53,6 +54,54 @@ class MeasurementRow extends FitnessItemRow {
   }
 }
 
+class MeasurementDateDialog extends StatefulComponent {
+  MeasurementDateDialog({ this.navigator, this.previousDate }) {
+    _selectedDate = previousDate;
```

Comment:
Yeah I wasn't sure if this qualified as "non-trivial work" but on reflection I agree that it's better to put it there.